### PR TITLE
Update wgpu to 0.15.1

### PR DESCRIPTION
**Pull request by larsjarlvik**
Updates wgpu to 0.15.1, tested with the examples and in my project.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ readme = "README.md"
 resolver = "2"
 
 [dependencies]
-wgpu = "0.14"
+wgpu = "0.15.1"
```
Comment (andreban) [inline Cargo.toml:15]: I'd just update the minor version, and leave the patch out. APIs shouldn't change between patches, so it should be fine.

```suggestion
wgpu = "0.15"
```

File: examples/hello.rs [hunk 2]
```diff
@@ -13,8 +13,15 @@ fn main() -> Result<(), Box<dyn Error>> {
...
-    let surface = unsafe { instance.create_surface(&window) };
+    let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
+        backends: wgpu::Backends::all(),
+        ..Default::default()
+    });
+    let surface = unsafe {
+        instance
+            .create_surface(&window)
+            .expect("Failed to create surface!")
+    };
```
Comment (andreban) [inline examples/hello.rs:24]: This function returns `Result<(), Box<dyn Error>>`. Wouldn't using the `?` operator work?

```suggestion
    let surface = unsafe { instance.create_surface(&window)?};
```

File: examples/depth.rs [hunk 3]
```diff
@@ -15,8 +15,15 @@ fn main() -> Result<(), Box<dyn Error>> {
...
-    let surface = unsafe { instance.create_surface(&window) };
+    let instance = wgpu::Instance::new(wgpu::InstanceDescriptor {
+        backends: wgpu::Backends::all(),
+        ..Default::default()
+    });
+    let surface = unsafe {
+        instance
+            .create_surface(&window)
+            .expect("Failed to create surface!")
+    };
```
Comment (andreban) [inline examples/depth.rs:26]: This function returns `Result<(), Box<dyn Error>>`. Wouldn't using the `?` operator work?

```suggestion
    let surface = unsafe { instance.create_surface(&window)?};
```

File: Cargo.toml (hunk 1 above)
Comment (larsjarlvik) [reply Cargo.toml]: You're right, resolved.

**Comment by larsjarlvik** [comment]
Hi, thanks for the review, fixed the issues mentioned.

**Pull request closed without merging**